app/Models/User.php

```diff
@@ -60,7 +60,7 @@ public function setBirthdateAttribute($birthdate)
     public function setPhoneAttribute($phone)
     {
         // Skip mutator if attribute is null.
-        if (is_null($phone)) {
+        if (empty($phone)) {
```

Comment:
Yep, so an empty string will just not set an attribute on the model. Otherwise we were getting non-unique constraint errors when phone was an optional field.